From 398e3e73eee4f2430650bb9e217eeb6d50f893f1 Mon Sep 17 00:00:00 2001
From: Thomas Sibley <trs@bestpractical.com>
Date: Tue, 22 Jan 2013 15:26:06 -0800
Subject: [PATCH] Let list of headers used for one-time recipient suggestions
 be adjusted

Extensions or local customizations may want to add RT-Send-To or
Reply-To, or remove Bcc, From, etc.
---
 lib/RT/Attachment.pm | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/RT/Attachment.pm b/lib/RT/Attachment.pm
index 35387eb..54217b3 100644
--- a/lib/RT/Attachment.pm
+++ b/lib/RT/Attachment.pm
@@ -499,12 +499,14 @@ L<Email::Address> objects.
 
 =cut
 
+our @ADDRESS_HEADERS = qw(From To Cc Bcc RT-Send-Cc RT-Send-Bcc);
+
 sub Addresses {
     my $self = shift;
 
     my %data = ();
     my $current_user_address = lc $self->CurrentUser->EmailAddress;
-    foreach my $hdr (qw(From To Cc Bcc RT-Send-Cc RT-Send-Bcc)) {
+    foreach my $hdr (@ADDRESS_HEADERS) {
         my @Addresses;
         my $line = $self->GetHeader($hdr);
         
-- 
1.7.11.3

